File: Wiki/C/code/C_Programming_A_Modern_Approach/ch02_C_Fundamentals/projects/01.c
// Purpose: use `printf()` to display the following diagram in the terminal: {{{
//
//            *
//           *
//          *
//     *   *
//      * *
//       *
//}}}
// Reference: page 34 (paper) / 59 (ebook)

#include <stdio.h>

    int
main(void)
{
    printf("       *\n");
    printf("      *\n");
    printf("     *\n");
    printf("*   *\n");
    printf(" * *\n");
    printf("  *\n");

    return 0;
}

File: Wiki/lua/code/Programming_in_Lua_3rd_edition/ch02_Types_and_Values/exercises/01.lua
-- Purpose: What is the value of the expression `type(nil) == nil`?
-- You can use Lua to check your answer.
-- Can you explain this result?

-- Reference: page 18 (paper) / 37 (ebook)


-- `type(nil) == nil`  is `false`  because the  output of  `type()` is  always a
-- string.  The contents of that string might be `nil`, but it's still a string.
-- OTOH, `type(nil) == 'nil'` is `true`.
